src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,46 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val backupLocator = By.id("del_10001")
+
+    /**
+     * Makes sure backup service is disabled
```

Comment:
Shouldn't it be named `makeSureIsDisabled` then? Now Javadoc suggests one thing and the method name another.

Reply:
This JavaDoc is redundant, I will remove it. The method name is clear.

Reply:
It's a copy-paste from `RichTextEditorConfiguration`:
```
/**
 * Makes sure RTE is disabled, even if the feature doesn't exist at all.
 * Supports both enabled and disabled websudo.
 */
```
`ensureRteIsDisabledRegardlessOfJiraVersionOrWebsudo` would tell the whole story, but it's not worth reminding that to every reader, caller or stacktrace. It's ok to put fine details in the docs.